### ISSUE TITLE
How do the IBF Pipelines work? ("IBF Pipeline 101")

### ISSUE THREAD

**Issue by diderikvw** [rodekruis/IBF-system#1664]
As IBF System Develop
I want to have a basic understanding of the IBF Pipelines
So that I can support Phuoc where needed, maintaining the Pipelines and doing small changes and fixes, can be a "sparring partner" for Phuoc, and think together with him and others in the team, including the PO about optimal ways forward in changing and improving the pipelines and how they fit into the IBF System's Software Architecture.

This is a follow-up from: [Update the architecture diagram #1584](https://github.com/rodekruis/IBF-system/issues/1584) 

[Accompanying Post in Teams here](https://teams.microsoft.com/l/message/19:b262590b1cc34ade9bbdb584f0765b31@thread.skype/1729580488652?tenantId=d3ab9790-6ae2-4bd8-aa5e-02864483e7c7&groupId=48e3c654-ac7d-4abc-9c70-ad637fb0a85f&parentMessageId=1729580488652&teamName=510%20-%20Anticipatory%20Action&channelName=%5BRD%5D%20IBF%20system&createdTime=1729580488652).

Refinement 5 Nov 2024:
1. Plan a team session of 1 hour.
2. Phuoc does a "show&tell" of IBF Pipelines (infrastructure, code, processes, repos, ...) => Team only, not with Aki and/or Jacopo for this first session.
4. Q&A + discussion
5. Define follow-up items, which could be: show&tell by Aki and/or Jacopo for more details and knowledge transfer.

**Comment by diderikvw** [comment]
@BlaiseSelvan @p-phung @gulfaraz I created this Spike as follow-up.

**Comment by diderikvw** [comment]
First opportunity on a Tuesday I could find is 25 November. I sent a calendar invite.
@p-phung please let me know if you want to prep this meeting together.

**Comment by p-phung** [comment]
@diderikvw I would like to make use this meeting to discuss how data specialist role can be integrating in the IBF e.g as a bridge between the IBF team and data pipeline owners. Do you agree to include it there in the meeting?

**Comment by diderikvw** [comment]
@p-phung Yes, IMO no topic is off the table for this first meeting. It is very likely we will need to plan follow-up meetings. Also with people outside of the IBF System Team.

As a side note:
> between the IBF team and data pipeline owners
We as IBF Team are the IBF Pipeline owners. We agreed on that several weeks ago when we aligned on the product and the team. But the differences in understanding about what this means in responsibilities, accountability and practical work, are what we a.o. will talk about.

**Comment by p-phung** [comment]
@diderikvw for this meeting, I would like to invite @aklilu to show us the data pipeline for typhoon in the Philippines. The idea is that we share knowledge about the pipeline with SW and prepare for December/January period when we have low capacity in data (between as Aki, Jacopo and I are all away). Additionally, I would like to make use of this meeting to revisit our discussion earlier about data pipeline knowledge and explore what works for us

@aklilu I have forwarded you the invite for this meeting at Tue 26/11/2024 14:00. Could you join?

**Comment by diderikvw** [comment]
@p-phung  Thank you for bringing this to my attention. I do not want Aki to participate in this first meeting, but first get our bearings as team. Also see Refinement. The meeting is just 1 hour and we will not be able to cover any complex topic extensively, so follow-up meetings will be needed anyway. @aklilu , nothing personal toward you. I know you are very knowledgeable about the pipelines. We will follow up with many more hours of meetings to do knowledge transfer in the coming months.

Good to know that you, Jacopo and Aki are unavailable in December and January. Do you know if all three of you are away for the full 2 months? This indeed requires us to look at knowledge transfer of IBF Pipelines a different way.

cc @BlaiseSelvan

**Comment by diderikvw** [comment]
Follow-up Spikes created, see [here in Teams](https://teams.microsoft.com/l/message/19:b262590b1cc34ade9bbdb584f0765b31@thread.skype/1733215990198?tenantId=d3ab9790-6ae2-4bd8-aa5e-02864483e7c7&groupId=48e3c654-ac7d-4abc-9c70-ad637fb0a85f&parentMessageId=1729580488652&teamName=510%20-%20Anticipatory%20Action&channelName=IBF%20-%20System&createdTime=1733215990198).
